docs/features/execution_payload_retention.md

```diff
@@ -4,7 +4,7 @@ title: Execution Payload Retention
 hide_title: false
 ---
 
-In addition to CloudWatch logs and AWS StepFunction API records, Cumulus automatically stores the initial and 'final' (the last update to the execution record) payload values as part of the Execution record in DynamoDB and Elasticsearch.
+In addition to CloudWatch logs and AWS StepFunction API records, Cumulus automatically stores the initial and 'final' (the last update to the execution record) payload values as part of the Execution record in your RDS database and Elasticsearch.
```

Comment:
Technically still stored in Dynamo and RDS

Reply:
Like my previous comment, I don't know that calling out Dynamo here is super valuable to the user.